specification/oep/resource-manager/readme.go.md

```diff
@@ -0,0 +1,26 @@
+## Go
```

Comment:
I believe this is a new RP therefore we need to change the configuration for go from track 1 (current) to the track 2 version. Please refer this document as example: https://github.com/Azure/azure-rest-api-specs/blob/main/documentation/samplefiles/readme.go.md

Please replace the content of this file entirely with the content in the above sample. In that sample `[[ServiceName]]` is a variable, and in your case, you need to use `oep` to replace it.